srcpkgs/swww/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'swww'
+pkgname=swww
+version=0.7.2
+revision=1
+build_style=cargo
+hostmakedepends="rust cargo"
+makedepends="scdoc pkg-config libxkbcommon-devel"
+depends="lz4"
+short_desc="Efficient animated wallpaper daemon for wayland, controlled at runtime"
+maintainer="Posi+ive <faizudinproman@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Horus645/swww"
+changelog="https://github.com/Horus645/swww/blob/main/CHANGELOG.md"
+distfiles="https://github.com/Horus645/swww/archive/refs/tags/v${version}.tar.gz"
+checksum=bb961577387aefd64b54090b3a7d0db5a393120edc24fe71d149e77fe132025c
+
+do_install() {
+	vbin target/${XBPS_TARGET_MACHINE}-unknown-linux-gnu/release/swww
+	vbin target/${XBPS_TARGET_MACHINE}-unknown-linux-gnu/release/swww-daemon
+	vman doc/generated/swww-clear.1
+	vman doc/generated/swww-daemon.1
+	vman doc/generated/swww-img.1
+	vman doc/generated/swww-init.1
+	vman doc/generated/swww-kill.1
+	vman doc/generated/swww-query.1
+	vman doc/generated/swww.1
```

Comment:
```suggestion
post_install() {
	for f in doc/generated/*.?; do
		vman $f
	done
```
the bins should be installed by the default do_install for the cargo build style

Reply:
The binaries aren't installed if I omit the vbin function. I have tested it twice and it doesn't install the binaries. Probably because there's `vcompletion` in `do_install`? Should I put that somewhere else?

Reply:
yes, look at what I posted, the function should be `post_install` not `do_install`

Reply:
Well, it packages. But it only packages the `swww` binary without its daemon counterpart. So I need to explicitly specify the daemon binary that I wanted to install like so:

```suggestion
do_install() {
	vbin target/${XBPS_TARGET_MACHINE}-unknown-linux-gnu/release/swww
	vbin target/${XBPS_TARGET_MACHINE}-unknown-linux-gnu/release/swww-daemon
```
unless there's another way that I didn't already knew?

Reply:
just add the second vbin to post_install, also use XBPS_RUST_TARGET

Reply:
Okay I have fixed the template, and added the zsh autocompletion file too. It is working now.